### PR TITLE
Fix next basics

### PR DESCRIPTION
**Pull request by TheOtterlord**
## Changes

Adds missing layout CSS. Doesn't add the gradient back because the full title is a prop.

## Testing

N/A

## Docs

N/A

### CODE REVIEW

File: examples/basics/src/layouts/Layout.astro [hunk 1]
```diff
@@ -11,3 +11,32 @@
 		<slot />
 	</body>
 </html>
+
+<style is:global>
```
Comment (Princesseuh) [inline examples/basics/src/layouts/Layout.astro:15]: Hmm, it'd be great if the vast majority of the styling was in the Welcome component, is that possible?

Comment (TheOtterlord) [reply examples/basics/src/layouts/Layout.astro:15]: Sure, it's already global, so it should just work.

**Review by Princesseuh — APPROVED** [review]
It's not super optimal, but since I'll be reworking it soon, as long as it works right now it's fine with me

**Comment by Princesseuh** [comment]
ah, the test catches what Chris brought up regarding styles leaking. We'll rework the template to use a direct import instead

**Comment by delucis** [comment]
Closing as I believe this has been superseded by #12472!

**Pull request closed without merging**